crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -43,8 +43,8 @@ use super::{Deferred, Resource, SystemBuffer, SystemMeta};
 /// }
 /// ```
 pub trait Command: Send + 'static {
-    /// Executes this command.
-    fn write(self, world: &mut World);
+    /// Applies this command.
```

Comment:
```suggestion
    /// Applies this command, causing it to mutate the provided `world`.
    ///
    /// This method is used to define what a command "does" when it is ultimately applied.
    /// Because this method takes `self`, you can store data or settings on the type that implements this trait.
    /// This data is set by the system or other source of the command, and then ultimately read in this method.
```